vividus-extension-selenium/src/main/java/org/vividus/selenium/manager/GenericWebDriverManager.java

```diff
@@ -146,6 +146,7 @@ public boolean isIOS()
     public static boolean isIOS(Capabilities capabilities)
     {
         return isPlatformName(capabilities, MobilePlatform.IOS)
+                || isPlatformName(capabilities, MobilePlatform.TVOS)
```

Comment:
isPlatformAnyOf(...) ?

Reply:
two is not too many to do refactoring

Reply:
![image](https://user-images.githubusercontent.com/17816729/112166371-9e615680-8c00-11eb-8784-88e20edc8709.png)